### DISCUSSION TITLE
Set consumed energy in Nous A1Z / TuYa TS011F to custom value

### DISCUSSION THREAD

**Discussion by mariushelf** [Koenkk/zigbee2mqtt#13482] [Devices]
Hi all, is there a way to set the consumed energy in the Nous A1Z / TuYa TS011F to a custom value?
I want to replace some faulty plugs which have already a lot of energy statistics. When replacing the fault plug with a new plug with zero consumed energy, it messes up my statistics like energy consumed this year etc.

I know I can set the consumed power to zero as described on the [device page](https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html). But is it possible to set a custom value?

I'm using home assistant btw, so a HA based solution is also welcome :)

Thanks a lot!

**Comment by nh2** [comment]
Same problem here, did you find a solution so far?
